pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -3074,6 +3074,11 @@ public boolean isAllowAutoSubscriptionCreation(final TopicName topicName) {
     }
 
     private AutoSubscriptionCreationOverride getAutoSubscriptionCreationOverride(final TopicName topicName) {
+        Optional<TopicPolicies> topicPolicies = getTopicPolicies(topicName);
```

Comment:
We should get `AutoSubscriptionCreationOverride` from `AbstractTopic#topicPolicies` I think

Reply:
Thanks for your reply :)
What's the difference between the way you provide and the above(`Line 3077`)?

Reply:
But there is no instance of `AbstractTopic` in `BrokerService`, so it can not get the `AutoSubscriptionCreationOverride`  from `AbstractTopic#topicPolicies` directly in `BrokerService`.